packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dedicatedCloud-datacenter.controller.js

```diff
@@ -24,6 +24,7 @@ export default class {
       )
       .then((data) => {
         this.isEOLDatacenter = COMMERCIAL_RANGE_NAME_EOL.includes(data);
+        this.datacenter.model.commercialRangeName = data;
```

Comment:
As the result is only used for a comparison, you should just bind a boolean to the controller
```suggestion
        this.datacenter.model.isNsxCommercialRange = data === VDC_TYPE.NSX;
```